<a href="https://colab.research.google.com/github/Rogeort/cinematica-particula/blob/main/main.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# PRACTICA 1. MOVIMIENTO RECTILINEO

## Problema 1. [solucion]

Se tiene que la ecuacion 1.
$$x(t) = \frac{2}{5} \sin{2 \pi t}  \text{ $m$}$$
es la posicion del punto $P$ en cualquier tiempo arbitrario $t$, por tanto su velocidad $v(t)$ sera su derivada respecto a $t$

In [ ]:
% Crear variable simbólica
syms t;

% Definir la función
x = (2/5) * sin(2*t*pi);

% Calcular la derivada
v = diff(x, t);

% Mostrar la derivada
disp(v);

por tanto $$ v(t) = \frac{4\,\pi \,\cos \left(2\,\pi \,t\right)}{5} \text{ $m$/$s$} $$

Evaluando en $t=0.375$

In [ ]:
% Evaluando en t=0.375
t_1 = subs(v, t, 0.375);
t_1 = double(t_1);

disp(t_1)

   -1.7772



luego $$ v(0.375) = -1.7772  $$

In [ ]:
% Derivando de nuevo respecto a t para obtener la aceleracoin
a = diff(v,t);
disp(a);

$$ a(t) = -\frac{8\,\pi^2 \,\sin \left(2\,\pi \,t\right)}{5}  \text{ $m$/$s^2$} $$

Evaluando en $t = 0.375$

In [ ]:
%evaluando
t_0 = 0.375;
t_1 = subs(a, t, t_0);
% numerico
t_1 = double(t_1);

disp(t_1)

  -11.1662



$$ a(0.375) = -11.1662 \text{ $m$/$s^2$} $$

# Problema [Solucion]. Intervalo $0\leq t \leq 2 $

## Grafica de $x(t)$

In [ ]:

% Definir el rango de tiempo
t = linspace(0, 2, 1000);

% Definir la función x(t)
x = (2/5) * sin(2 * pi * t);

% Graficar la función
figure;
plot(t, x, 'b', 'LineWidth', 2);
xlabel('t');
ylabel('x(t)');
title('Gráfica de x(t) = (2/5) sin(2\pi t)');
grid on;

## Grafica $v(t)$ en el intervalo

In [ ]:


% Definir la función v(t)
v = (4 * pi / 5) * cos(2 * pi * t);

% Graficar la función
figure;
plot(t, v, 'r', 'LineWidth', 2);
xlabel('t');
ylabel('v(t)');
title('Gráfica de v(t) = (4\pi / 5) \cos(2\pi t)');
grid on;

## Grafica $a(t)$ en el intervalo

In [ ]:



% Definir la función a(t)
a = -(8 * pi^2 / 5) * sin(2 * pi * t);

% Graficar la función
figure;
plot(t, a, 'g', 'LineWidth', 2);
xlabel('t');
ylabel('a(t)');
title('Gráfica de a(t) = -(8\pi^2 / 5) \sin(2\pi t)');
grid on;